### PR TITLE
Add Release Notes to Figgy

### LINKED ISSUE #3225: Create a changelog
with process

Requested at ADAPT+ meeting as well as github presentation.

### LINKED ISSUE #3228: CI should fail a step if we don't add to release notes
This step shouldn't block us from merging, but should force us to think about if something should be added there.

### PR DESCRIPTION
**Pull request by tpendragon**
Closes #3225 
Closes #3228 
Screen Shot 2019-07-26 at 5 48 16 PM

### CODE REVIEW

File: app/assets/stylesheets/components/header.scss [hunk 1]
```diff
@@ -258,3 +254,18 @@ header {
...
+  }
+
+  .menu--level-1>ul {
+    display: inline-block;
+
+    @media (min-width: 645px) {
+      margin-top: 0;
+      @include vertical-align;
+    }
+  }
```
Comment (hackmastera) [inline app/assets/stylesheets/components/header.scss]: do we still need the changes in this file with the lux header?

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -1,5 +1,7 @@
 <% provide :body do %>
-<%= render 'shared/header' %>
+<div class="lux">
```
Comment (hackmastera) [inline app/views/layouts/application.html.erb]: @sdellis we added a second lux load... is this bad? There was one already for the order manager

File: config/initializers/extra_extra.rb [hunk 3]
```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+ExtraExtra.renderer_options = {
+  header_level_offset: 1, # h1's are h1
```
Comment (hackmastera) [inline config/initializers/extra_extra.rb]: My understanding is that the only `h1` should be your home link, and page headings should start at h2.

**Comment by hackmastera** [comment]
@tpendragon if you want me to mess with this further I'm happy to; I know you're on another sprint this week.

File: app/assets/stylesheets/components/header.scss (hunk 1 above)
Comment (tpendragon) [reply app/assets/stylesheets/components/header.scss]: We don't need the ul bit, but do need to fix the height

File: config/initializers/extra_extra.rb (hunk 3 above)
Comment (tpendragon) [reply config/initializers/extra_extra.rb]: Fixed.

File: app/assets/stylesheets/components/header.scss (hunk 1 above)
Comment (tpendragon) [reply app/assets/stylesheets/components/header.scss]: Fixed.

File: app/views/layouts/application.html.erb (hunk 2 above)
Comment (tpendragon) [reply app/views/layouts/application.html.erb]: They shouldn't run into each other, but yeah @sdellis would know better.

Comment (hackmastera) [reply app/views/layouts/application.html.erb]: 👍 confirmed via slack: "yes you can load more than one lux mounting element. I think there may be an issue with multiple mounting elements  in the static build"

**Pull request merged**